content/articles/goodbye-2019-hello-2020.md

```diff
@@ -0,0 +1,25 @@
+Title: Goodbye 2019! Hello, 2020!
+Date: 12-23-2019
+Tags: updates, blog
+Slug: goodbye-2019-hello-2020
+Authors: Omar Quimbaya
+Summary: A reflection of our community in 2019
+
+
+Andrew Herrington, Daniel Wallce, Doug Mendizabal, and I started this group on July 24, 2017. The idea to create a slack community of developers from around San Antonio came from a developer named Mark. He had recently come in from Tennessee and reached out to Omar via the Geekdom Slack. He was looking for a place where developers could share ideas and be language agnostic, and in Nashville, they had a slack community with developers from all over the city. Why didn't San Antonio have that?
+
+Well, we had the Geekdom slack, but not everyone who was a developer has a $50 a month membership to that coworking space, so the cost alone was prohibitive, and the slack group itself was declining in terms of its community participation. There had to be something better, so the four banded together and worked on building a developer community. 
+
+Ever since then, our community has grown from those 5 developers (Mark included) to 474 members. Of those 474 members, 71 are currently active, as of writing this blog post. We have not been tracking a lot of metrics from our slack group since its inception, but over the course of two years, we think this is pretty good. 
+
+What are the goals of the community? Developers to connect, help each other out, and grow as professionals in their craft. The way we support this is by making sure the slack community is an inclusive one where people of all walks of life are welcome; however, there are opportunities for us to improve on this. We welcome community feedback, as well, to help us improve as admins. Please feel free to reach out to us in the DMs if you have feedback or questions. We also support the goals by having quaterly meetups featuring talks from non-technical topics to technical topics. This helps people be introduced to new ideas, develop their soft-skills, meet new people by putting a face to a screenname, and feed them for an evening. These meetups would not have happened without the help of Jon Robers in our group. He took up the sword of event development, organized the admin team, and created some great meetups throughout the year. We are very grateful for the time and work he had put into making those meetups successful.
+
+Our community also started two open source projects: Barbacoa and PuroEvents. People within our community have the opportunity to dive in and contribute to those projects as they see fit, and that helps new developers get into open source software development, as well. It's a great way for experienced developers to mentor new people, too, so the opportunity is there for everyone.
```

Comment:
I think this is disingenuous, we kind of started barbacoa, but never really did anything, and PuroEvents only has a readme and nothing else.

Reply:
I disagree. It's a start of something. The thing that needs to be done is organizing people and getting them involved. If you write open source code and no one knows about it, is it really open source?